# No recibe nada
# std_msgs/Empty
---
# Retorna:
# PointCloud
sensor_msgs/PointCloud2 pointcloud
# Informaci√≥n de error
memoria/ErrorMsg error